contracts/script/upgrade/upgrade_mainnet_v1_v2/DEPLOY.md

```diff
@@ -0,0 +1,74 @@
+# V1 to V2 Upgrade Process
```

Comment:
should the versioning be in-alignment with what we're using on NPM? https://www.npmjs.com/package/@eigenda/contracts